src/ui/patch_hub/series_details.sh

```diff
@@ -1,102 +1,63 @@
 include "${KW_LIB_DIR}/ui/patch_hub/patch_hub_core.sh"
 
-# Screen resposible for show a specific patch details
+# This function is responsible for showing the screen with a patchset details
+# and actions available. The patchset can be a bookmarked one (in case
+# `@show_bookmarked_patchset` is set) or from a list of patchsets passed as
+# argument (`@_target_patch_metadata`).
 #
-# @patch_index: Patch index
+# @series_index: Series index
 # @_target_patch_metadata: List with patches metadata
+# @show_bookmarked_patchset: If set, the patchset to be shown is a bookmarked
+#   one, i.e., the details will be fetch in local database not in
+#   `@_target_patch_metadata`.
 function show_series_details()
 {
-  local patch_index="$1"
-  local -n _target_patch_metadata="$2"
+  local series_index="$1"
+  local -n _target_patch_metadata
+  local show_bookmarked_patchset="$3"
```

Comment:
I don't get this part. Is the second argument of this function irrelevant? If so, wouldn't it be neater to rearrange things as to have just 2 arguments, instead of 3?

Reply:
Hi @magalilemes.
Really sorry for not responding 😨.
Thanks for the heads up. Fixed it.